src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1033,19 +1033,19 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
             const float dy = newTouchPoint.Y - anchor.Y;
 
-            // If we've moved more than one row of text, we'll want to scroll the viewport
-            if (std::abs(dy) > fontHeight)
+            // Start viewport scroll after we've moved more than a half row of text
+            if (std::abs(dy) > (fontHeight / 2.0f))
```

Comment:
Use safemath on the division

Reply:
Hi @carlos-zamora, I saw that the Chromium safe math headers are already in the `TermControl.cpp`'s `#include` (as part of `LibraryIncludes.h`), and I can just use those by using the `::base` namespace.
However, I am sorry for my ignorance, but why do we need to clamp this division, which is performed against a constant value (in this case 2.0f)?

Reply:
Sorry! Force of habit. This one probably doesn't need it...